src/components/NewsletterSignup/NewsletterSignup.tsx

```diff
@@ -0,0 +1,206 @@
+  /** Optional: Appears below the input field's example text to provide any additional instructions. Accepts a string or
+   *  an element. */
+  formHelperText?: string | JSX.Element;
+  /** Optional: ID that other components can cross-reference for accessibility purposes */
+  id?: string;
+  /** Optional: Toggles the invalid state for the email field. */
+  isInvalidEmail?: boolean;
+  /** Optional: Value to determine the section color highlight */
+  newsletterSignupType?: SectionTypes;
+  /** Required: A handler function that will be called when the form is submitted. */
+  onSubmit: (event: React.FormEvent<any>) => void;
+  /** Required: A handler function that will be called when the text input changes. */
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  /** Optional: Used to populate the `<h3>` header title. */
+  title?: string;
+  /** Optional: The value of the email text input field. */
+  valueEmail?: string;
+  /** Optional: Used to specify what is displayed in the component form/feedback area. */
+  view?: "form" | "submitting" | "confirmation" | "error";
+}
+
+const defaultDescriptionText =
+  "Stay connected with the latest research news from NYPL, including information about our events, programs, " +
+  "exhibitions, and collections.";
+
+/**
+ * The NewsletterSignup component provides a way for patrons to register for an
+ * email-based newsletter distribution list.
+ */
+export const NewsletterSignup = chakra(
+  forwardRef<HTMLDivElement, NewsletterSignupProps>(
+    (
+      {
+        className,
+        confirmationHeading,
+        confirmationText,
+        descriptionText = defaultDescriptionText,
+        formHelperText,
+        id,
+        isInvalidEmail = false,
+        newsletterSignupType = "whatsOn",
+        onChange,
+        valueEmail,
+        onSubmit,
+        title = "Sign Up for Our Newsletter!",
+        view = "form",
+        ...rest
+      },
+      ref?
+    ) => {
+      const { isLargerThanMobile } = useNYPLBreakpoints();
+      const styles = useStyleConfig("NewsletterSignup", {
+        newsletterSignupType,
+      });
+      const iconColor = useColorModeValue(null, "dark.ui.typography.body");
+
+      const isFormView = view === "form" || view === "submitting";
+
+      // Manage focus to ensure accessibility when confirmation or error message is rendered.
+      const focusRef = React.useRef<HTMLDivElement>(null);
+      // When view prop changes, set focus to the confirmation or error content element.
+      React.useEffect(() => {
+        focusRef.current?.focus();
+      }, [view]);
+
+      return (
+        <Stack
+          direction={isLargerThanMobile ? "row" : "column"}
+          ref={ref}
+          __css={styles}
+          {...rest}
+        >
+          <VStack id="pitch">
+            <Heading level="h3" text={title} />
+            <Text>{descriptionText}</Text>
+            <Link
+              // @TODO I would make this a prop -- WL.
```

Comment:
The url or the entire link component? I agree if it's the url because it _could_ change and the DS should not be responsible for the update. If this is a prop, the default can be `"https://www.nypl.org/help/about-nypl/legal-notices/privacy-policy"`.

@bigfishdesign13 this sounds like a good update to the spec doc and component.

Reply:
@nypl-wluisi , sorry forgot to tag initially.

Reply:
I added an optional prop called `privacyPolicyLink` of type `string`.